Week05/lesson10/classwork/city_changer_starter_code/js/index.js

```diff
@@ -0,0 +1,53 @@
+// User types in a city name
+// User clicks on Update button to submit
+// Word is matched with an image
+// Background image changes accordingly
+
+
+// Use the below to prevent the default behavior of the browser which in this case is to refresh the current page.
+//
+// function switchCity(event){
+//   event.preventDefault();
+// }
+//
+
+
+function switchCity(event) {
+
+  event.preventDefault();
+
+  var city = document.querySelector("#city-type").value;
```

Comment:
Good shout, funnily enough I actually did both of those things when I got into the cash registrer HW.